atomic_reactor/plugins/post_compress.py

```diff
@@ -47,7 +47,11 @@ def __init__(self, tasker, workflow, load_exported_image=False, method='gzip'):
                                     you may load the exported tar with this switch
         """
         super(CompressPlugin, self).__init__(tasker, workflow)
-        self.load_exported_image = load_exported_image
+        if base_image_is_scratch(self.workflow.builder.base_image.to_str()):
+            self.log.debug("build based on scratch image, do not load exported image")
+            self.load_exported_image = False
+        else:
+            self.load_exported_image = load_exported_image
```

Comment:
Removed
- Do not squash images that have scratch base
- Do not load exported image when basing on scratch
from the first commit and cleaned history.

Now we utilize git history to find the bottom layer of built image and squash from that. Is this viable?